### DISCUSSION TITLE
Listening is not available in your country

### DISCUSSION THREAD

**Discussion by Saxalinpro** [ReVanced/revanced-discussions#1030] [General]
Is there a patch that bypasses this problem?

![Screenshot_2024-03-04-01-58-51-371_app.revanced.android.apps.youtube.music.jpg](https://github.com/ReVanced/revanced-discussions/assets/154323367/8ac68e5a-945f-43dc-be42-76ab0c82f28e)

**Comment by oSumAtrIX** [comment]
You can see the list of patches in ReVanced Manager

**Reply by Saxalinpro** [reply]
Why not make a patch that will exempt you from regional sanctions?

**Reply by oSumAtrIX** [reply]
For the same reason why 1+1 doesn't add up to anything other than 2. Unless there's a way, late YouTube versions use your IP to get your location.
